examples/notificationAsync.c

```diff
@@ -115,13 +135,22 @@ int main (void)
 	ElektraIoTimerOperation * timer = elektraIoNewTimerOperation (2000, 1, printVariable, &value);
 	elektraIoBindingAddTimer (binding, timer);
 
-	kdbGet (kdb, config, key);
-
 	printf ("Asynchronous Notification Example Application\n");
 	printf ("- Set \"%s\" to red, blue or green to change the text color\n", keyName (callbackKeyToWatch));
 	printf ("- Set \"%s\" to any integer value\n", keyName (intKeyToWatch));
 	printf ("Send SIGINT (Ctl+C) to exit.\n\n");
 
+	// Get configuration
+	kdbGet (kdb, config, key);
+	printVariable (timer); // "value" was automatically updated
+
+	// This allows us to wake the loop from our signal handler
+#ifdef HAVE_LIBUV1
```

Comment:
@ingwinlu do we have libuv1 installed somewhere?

Reply:
libuv 1.x was included in the debian docker images, but afaik they are not used by the build server.
On Travis we only have libuv 0.x available. This PR will add 1.x to the build server.